src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -341,6 +498,51 @@ public override TResult Accept<TResult>(SymbolVisitor<TResult> visitor)
             return visitor.VisitTupleType(this);
         }
 
+        internal override AttributeUsageInfo GetAttributeUsageInfo()
+        {
+            throw new NotImplementedException();
+        }
+
+        internal override ImmutableArray<Symbol> GetEarlyAttributeDecodingMembers()
+        {
+            throw new NotImplementedException();
+        }
+
+        internal override ImmutableArray<Symbol> GetEarlyAttributeDecodingMembers(string name)
+        {
+            throw new NotImplementedException();
+        }
+
+        internal override NamedTypeSymbol GetDeclaredBaseType(ConsList<Symbol> basesBeingResolved)
+        {
+            throw new NotImplementedException();
```

Comment:
it should be same as the base of the underlying type. We will not use base for the purpose of lookup, but may still need the base when analyzing casting.